glean-core/src/storage/mod.rs

```diff
@@ -129,6 +129,7 @@ impl StorageManager {
         storage: &Database,
         store_name: &str,
         metric_id: &str,
+        metric_lifetime: Lifetime,
```

Comment:
Looking at this again I wonder if it would be better to just pass &CommonMetricData directly to this function 🤔

Reply:
But then we would also need to pass `glean` because otherwise you won't get the identifier (`self.meta().identifier(glean)`). But `StorageManager` shouldn't need to know about Glean, it should only know about the database.

Reply:
Ah! You are right, better this way then.